### PR TITLE
Add docs for `$fail` action

### PR DESCRIPTION
**Pull request by shay2025**
This pull request adds documentation for the `$fail` action builtin.

### CODE REVIEW

File: aladino/builtins.md [hunk 1]
```diff
@@ -1670,6 +1670,43 @@ workflows:
       - $error("This pull request was considered too large.")
 ```
 
+
+## &nbsp; fail
+______________
+
+**Description**:
+
+Fails the action with a given fail message.
```
Comment (marcelosousa) [inline aladino/builtins.md:1679]: I think we should be explicit where this fail message will appear. It is in the logs of the GH action right?

**Review by marcelosousa — CHANGES_REQUESTED** [review]
Please check the comment!

File: aladino/builtins.md (hunk 1 above)
Comment (shay2025) [reply aladino/builtins.md:1679]: Yes

**Review by ferreiratiago — APPROVED** [review]
LGTM

**Pull request merged**